app-crypt/moolticute/files/moolticuted.init

```diff
@@ -1,11 +1,11 @@
 #!/sbin/openrc-run
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
2023

Reply:
Looks like i missed one. Should be fixed now